pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -329,19 +352,24 @@ func (dsw *desiredStateOfWorld) GetVolumesToMount() []VolumeToMount {
 				volumesToMount,
 				VolumeToMount{
 					VolumeToMount: operationexecutor.VolumeToMount{
-						VolumeName:          volumeName,
-						PodName:             podName,
-						Pod:                 podObj.pod,
-						VolumeSpec:          podObj.spec,
-						PluginIsAttachable:  volumeObj.pluginIsAttachable,
-						OuterVolumeSpecName: podObj.outerVolumeSpecName,
-						VolumeGidValue:      volumeObj.volumeGidValue,
-						ReportedInUse:       volumeObj.reportedInUse}})
+						VolumeName:            volumeName,
+						PodName:               podName,
+						Pod:                   podObj.pod,
+						VolumeSpec:            podObj.spec,
+						PluginIsAttachable:    volumeObj.pluginIsAttachable,
+						OuterVolumeSpecName:   podObj.outerVolumeSpecName,
+						VolumeGidValue:        volumeObj.volumeGidValue,
+						ReportedInUse:         volumeObj.reportedInUse,
+						OperationCompleteFunc: volumeObj.operationCompleteFunc}})
 		}
 	}
 	return volumesToMount
 }
 
+func (dsw *desiredStateOfWorld) GetVolumeUnmountMetric(volumeName v1.UniqueVolumeName) func(error) {
```

Comment:
I think we should rename this function because it does not exactly return the metric itself, but it returns a callback which can be called to emit the metric.